tasks/utils.py

```diff
@@ -29,37 +104,7 @@ def load_manifest(mtype, context = {}):
     manifest_template = yaml.safe_load(mt.read())
     return jsone.render(manifest_template, context)
 
-def k8s_apply(manifest, dry_run = True):
-  """run `kubectl apply` on the specified manifest"""
-  # write out the modified config
-  with tempfile.NamedTemporaryFile(mode='w') as tf:
-    yaml.dump(manifest, tf)
-    tf.flush()
-
-    if dry_run:
-      print("The config we generated:")
-      print(open(tf.name).read())
-      run('kubectl apply --dry-run -f %s' % tf.name)
-
-    else:
-      run('kubectl apply -f %s' % tf.name)
-
-def get_pods():
-  """List running pods"""
-  items = json.loads(
-    run('kubectl get pods -l app=pyspaceship -o=json', hide=True).stdout
-  )['items']
-
-  pods = []
-  for item in items:
-    pods.append({
-      'name': item['metadata']['name'],
-      'image': item['status']['containerStatuses'][0]['image'],
-      'phase': item['status']['phase'],
-    })
-
-  return pods
-
-def random_pod_name():
-  return [p['name'] for p in get_pods() if p['phase'] == 'Running'].pop()
+def random_string(length):
+  return "".join( random.SystemRandom().choices(string.ascii_letters + string.digits, k=length))
```

Comment:
Same difference, but python 3.6 added a "secrets" module that also exposes SystemRandom.